Automattic/wp-calypso
# Jetpack Cloud Pricing page | Update Jetpack Complete description

Jetpack Complete includes Social Basic with unlimited shares but the description on the pricing page says, "Social: Basic with 1,000 shares/mo". This PR updates the description to reflect the change.

#### Proposed Changes

* Update description for **Security** and **Complete** comparison table
* Update description of **Jetpack Complete** inside lightbox

#### Testing Instructions

- Do any one of these
    - Click on Jetpack Cloud live link below and wait for the redirect to complete, then go to `/pricing`
    - or boot up this PR 
        - Run `git fetch && git checkout fix/jetpack-complete-description`
        - Run `yarn start-jetpack-cloud`
        - Goto [http://jetpack.cloud.localhost:3000/pricing](http://jetpack.cloud.localhost:3000/pricing)
- Under the comparison table for Security and Complete, confirm that you see "Social: Basic with unlimited shares" instead of "Social: Basic with 1,000 shares/mo" under the **Growth** section.
- For the Complete plan, click on the **More about Complete** link to open the lightbox
- Confirm that you see "Social: Basic with unlimited shares" instead of "Social: Basic with 1,000 shares/mo" under the **Includes** section.


#### Pre-merge Checklist





Fixes 1198218726984184-as-1203245280300155/f

| BEFORE | AFTER |
| - | - |
| <img width="1328" alt="Screenshot 2022-11-22 at 1 11 05 PM 1" src="https://user-images.githubusercontent.com/18226415/203257480-9a02cf3d-c7f4-45b2-82cf-dbb33b1294ae.png"> | <img width="1320" alt="Screenshot 2022-11-22 at 1 14 35 PM" src="https://user-images.githubusercontent.com/18226415/203257506-b2a94cfb-8928-420e-bad6-5646420430c0.png"> |
| <img width="1096" alt="Screenshot 2022-11-22 at 1 15 30 PM" src="https://user-images.githubusercontent.com/18226415/203257589-445e84d9-dc85-4998-b816-3253a82addee.png"> | <img width="1100" alt="Screenshot 2022-11-22 at 1 16 05 PM" src="https://user-images.githubusercontent.com/18226415/203257656-9c8c8265-42b9-4fd9-94de-e23ed67a5cf6.png"> |

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203245280300155
